olive/engine/footprint.py

```diff
@@ -64,6 +64,13 @@ def __init__(
     def record_objective_dict(self, objective_dict):
         self.objective_dict = objective_dict
 
+    def _is_empty_metric(self, metric: FootprintNodeMetric):
+        if metric is None:
+            return True
+        if metric.value is None or len(metric.value) == 0:
```

Comment:
>metric.value is None or len(metric.value) == 0 [](http://example.com/codeflow?start=11&length=46)

[https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty](https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty)

Reply:
Done.